SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -149,6 +149,10 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "Wait Time Before Processing Received RPC With Unresolved Refs"))
 	float QueuedIncomingRPCWaitTime;
 
+	/** Seconds to wait before dropping an outgoing RPC, due to lack of authority over the Entity*/
```

Comment:
Is this the only reason we would drop them? Aren't there other ways for them to time out?

Reply:
You're right, that comment is outdated, will update.

Reply:
Updated.